examples/reverse_words/__init__.py

```diff
@@ -169,6 +173,19 @@ def main(mode, save_path, num_batches, data_path=None):
         for brick in model.get_top_bricks():
             brick.initialize()
 
+        # Give an idea of what's going on
+        model = Model(cost)
```

Comment:
These are not changes for this PR, which means that probably you should rebase your PR one more time (merge the master branch into it).